api/client/client.go

```diff
@@ -692,3 +692,53 @@ func (c *Client) GenerateDatabaseCert(ctx context.Context, req *proto.DatabaseCe
 	}
 	return resp, nil
 }
+
+// GetRole returns role by name
+func (c *Client) GetRole(name string) (types.Role, error) {
```

Comment:
Why not have the new APIs take the context as parameter?

Reply:
I agree, let's have all new endpoints take a context and propagate it.